tests/api/test_attributes.py

```diff
@@ -98,6 +98,32 @@ def test_attributes_query(user_api_client, product):
     assert len(attributes_data) == attributes.count()
 
 
+NOT_EXISTS_IDS_ATTRIBUTES_QUERY = """
+    query ($filter: AttributeFilterInput!) {
+        attributes(first: 5, filter: $filter) {
+            edges {
+                node {
+                    id
+                    name
+                }
+            }
+        }
+    }
+    """
+
+
+@pytest.mark.parametrize(
+    "attribute_filter", [{"ids": ["ygRqjpmXYqaTD9r=", "PBa4ZLBhnXHSz6v="]}],
+)
+def test_attribute_not_exit_ids(attribute_filter, user_api_client, category):
```

Comment:
I would change def's to `test_attributes_query_ids_not_exists`.

Reply:
done. thank you 👍